.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker Image CI
```

Comment:
Change the name of the workflow to something useful, like "Build Docker Image and Deploy to Google Cloud" or something.